src/Analysis/GWASResults/Views/Home/ManageColumns/ManageColumns.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useContext, useEffect } from 'react';
+import {
+  Button, Dropdown, Switch, message,
+} from 'antd';
+import RestoreIcon from './ManageColumnsIcons/RestoreIcon';
+import HolderIcon from './ManageColumnsIcons/HolderIcon';
+import ManageColumnsIcon from './ManageColumnsIcons/ManageColumnsIcon';
+import InitialColumnManagement from '../InitialState/DefaultColumnManagement';
+import SharedContext from '../../../Utils/SharedContext';
+import './ManageColumns.css';
+
+const ManageColumns = () => {
+  const { homeTableState, setHomeTableState } = useContext(SharedContext);
+
+  useEffect(() => {
+    // if using LocalStorage, update the columnManagement
+    // storage with each columnManagement update
+    if (homeTableState.useLocalStorage) {
+      localStorage.setItem(
+        'columnManagement',
+        JSON.stringify(homeTableState.columnManagement),
+      );
```

Comment:
can we move this interaction with the local storage closer to where we have the other one (in `const DetermineInitialColumnManagement = () => {` )? Maybe have an exported function there like `updateColumnManagement`?

Reply:
Good idea! 

I renamed the folder from InitialState to HomeTableState since the folder no long exclusively holds logic for intializing the state. I created a file UpdateColumnManagement.js that contains this function. This file lives alongside the DetermineIntialColumnManagement.js.